src/BasePlatform.js

```diff
@@ -79,10 +79,12 @@ export default class BasePlatform {
      * with getUserMedia, return a string explaining why not.
      * Otherwise, return null.
      */
-    screenCaptureErrorString() {
+    screenCaptureErrorString(): string {
         return "Not implemented";
     }
 
+    isElectron(): boolean { return false; }
```

Comment:
it could also just be a property and then accessed as `PlatformPeg.get().isElectron`
that way no other classes have to implement it but is also weird, I guess its all weird with the current settings architecture